web/src/search/input/SearchPage.tsx

```diff
@@ -121,8 +121,8 @@ export const SearchPage: React.FunctionComponent<Props> = props => {
                             </div>
                         ))}
                     </div>
-                    <div className="search-page__help-content mt-5">
-                        <div className="search-page__example-searches">
+                    <div className="search-page__help-content row mt-5">
+                        <div className="search-page__example-searches col-sm-12 col-md-12 col-lg-5 col-xl-6">
```

Comment:
Specifying the intermediate steps shouldn't be necessary, it should use the same size up to the next breakpoint:

> Grid breakpoints are based on minimum width media queries, meaning they apply to that one breakpoint and all those above it (e.g., .col-sm-4 applies to small, medium, large, and extra large devices, but not the first xs breakpoint).

https://getbootstrap.com/docs/4.5/layout/grid/

```suggestion
                        <div className="search-page__example-searches col-xs-12 col-lg-5 col-xl-6">
```